streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/StreamParkConsoleBootstrap.java

```diff
@@ -48,11 +52,23 @@
 @EnableScheduling
 public class StreamParkConsoleBootstrap {
 
+    @Autowired
+    private RegistryService registryService;
+
     public static void main(String[] args) throws Exception {
         new SpringApplicationBuilder()
             .properties(SpringProperties.get())
             .sources(StreamParkConsoleBootstrap.class)
             .run(args);
     }
 
+    @PostConstruct
+    public void init() {
+        registryService.start();
```

Comment:
We need to decide whether to call this method based on the configuration value "high-availability.enable."

Reply:
changed